pkg/platform/runtime/buildscript/file.go

```diff
@@ -36,6 +36,9 @@ func newScriptFromFile(path, org, project string, auth *authentication.Auth) (*S
 	logging.Debug("Build script does not exist. Creating one.")
 	commitId, err := localcommit.Get(filepath.Dir(path))
 	if err != nil {
+		if localcommit.IsFileDoesNotExistError(err) {
+			return nil, nil // headless project
+		}
```

Comment:
Headless projects still have commits.

Reply:
Not the very first one, e.g. `state install` in an empty directory. I forgot that case too.

Reply:
That might be an order of operations issue though. Even those projects have a commit, they just don't have a parent commit. I'm guessing this code gets invoked before we have created the commit? Is there any way we can change that so we don't need special handling here?

Reply:
Okay, I found it. When running `state install` in an empty directory, I made it so the commit ID file is written before the buildscript.